gno.land/cmd/gnoland/start.go

```diff
@@ -174,283 +189,153 @@ func execStart(c *startCfg, io *commands.IO) error {
 		cfg, loadCfgErr = config.LoadConfigFile(c.nodeConfigPath)
 	} else {
 		// Load the default node configuration
-		cfg, loadCfgErr = config.LoadOrMakeConfigWithOptions(rootDir, nil)
+		cfg, loadCfgErr = config.LoadOrMakeConfigWithOptions(dataDir, nil)
 	}
 
 	if loadCfgErr != nil {
 		return fmt.Errorf("unable to load node configuration, %w", loadCfgErr)
 	}
 
-	// create priv validator first.
-	// need it to generate genesis.json
-	newPrivValKey := cfg.PrivValidatorKeyFile()
-	newPrivValState := cfg.PrivValidatorStateFile()
-	priv := privval.LoadOrGenFilePV(newPrivValKey, newPrivValState)
-
-	// write genesis file if missing.
-	genesisFilePath := filepath.Join(rootDir, cfg.Genesis)
-
-	genesisTxs, genesisTxsErr := loadGenesisTxs(c.genesisTxsFile, c.chainID, c.genesisRemote)
-	if genesisTxsErr != nil {
-		return fmt.Errorf("unable to load genesis txs, %w", genesisTxsErr)
-	}
-
+	// Write genesis file if missing.
+	genesisFilePath := filepath.Join(dataDir, cfg.Genesis)
```

Comment:
```suggestion
	genesisFilePath := cfg.GenesisFullPath()
```

Could make sense to implement this helper.

Reply:
will do this in a next iteration!